### ISSUE TITLE
Unable to reproduce the results on MS COCO datasets

### ISSUE THREAD

**Issue by chenf99** [MegviiDetection/FSCE#24]
Dear author, I tried to reproduce the results on MS COCO using the default config and base model from [ http://dl.yf.io/fs-det/models/coco/base_model/model_final.pth]( http://dl.yf.io/fs-det/models/coco/base_model/model_final.pth), but  got average AP 10.460 on 10 seeds, could you please provide the model_final.pth you trained? Thank you very much!

**Comment by bsun0802** [comment]
How many GPUs did you use?

**Comment by chenf99** [comment]
> How many GPUs did you use?

4 with batchsize unchanged.

**Comment by bsun0802** [comment]
The specification we provide is for 8-GPUs.

**Comment by kike-0304** [comment]
> 我们提供的规格适用于 8-GPU。

Why do you need 8gpu? Can't I just keep the batch size consistent?

**Comment by bsun0802** [comment]
Batch size is a hyper parameter, just like learning rate. Few-shot learning is sensitive to training setting due to low data regime. 

You can experiments with 4 cards, but we didn't have recommendations for hyper-params for it.

A quick theoretical background on batch size and learning rate: https://stackoverflow.com/questions/53033556/how-should-the-learning-rate-change-as-the-batch-size-change
